test/utils/Nonces.test.js

```diff
@@ -0,0 +1,20 @@
+require('@openzeppelin/test-helpers');
+
+const NoncesImpl = artifacts.require('NoncesImpl');
+
+contract('Nonces', function (accounts) {
+  const [ sender ] = accounts;
+
+  beforeEach(async function () {
+    this.nonces = await NoncesImpl.new();
+  });
+
+  it('gets a nonce', async function () {
+    expect(await this.nonces.nonces(sender)).to.be.bignumber.equal('0');
+  });
+
+  it('increment a nonce', async function () {
```

Comment:
Let's add an assertion that the nonce for an "other" account is unaffected by `useNonce`. So we can check that the nonce is specific to the address argument.